models/volume_transmitter.h

```diff
@@ -48,30 +48,34 @@ Description
 +++++++++++
 
 The volume transmitter is used in combination with neuromodulated
-synaptic plasticty, plasticity that depends not only on the activity
+synaptic plasticity, plasticity that depends not only on the activity
```

Comment:
maybe you can rephrase this sentence to something like: The volume transmitter is used in combination with neuromodulated synaptic plasticty, this plasticity depends not only on the activity ...

it's just a small thing, I leave you the choice to ignore or accept this